corehq/ex-submodules/casexml/apps/case/templates/case/partials/case_hierarchy.html

```diff
@@ -124,7 +127,17 @@
             </div>
           {% endif %}
         </td>
-
+        {% if request|toggle_enabled:"SUPPORT" %}
+          <td>
+            {% if case.case_id == current_case.case_id %}
+              <em>(current case)</em>
+            {% elif case.index_info.is_ancestor %}
+              <strong>{{ case.index_info.identifier }}</strong> / {{ case.index_info.relationship }}
+            {% elif case.index_info %}
+              {{ case.index_info.identifier }} / <strong>{{ case.index_info.relationship }}</strong>
+            {% endif %}
```

Comment:
looking at the screenshot in the description I found it confusing that the bold text was different for different rows. It took a moment to figure out what it meant. I'm curious to hear why you formatted it like this.

Reply:
Good point, I wasn't sure how to convey that information visually.  Basically, the indices are "through" models in many-to-many relationships, so mapping them onto to the models was a little tricky.  I settled on only traversing indices one level (in both directions) from the current case, and storing the relationship information on the other case in each relationship.  However, a case could have a host/extension or parent/child relationship in either direction, and I wanted to spell out which was the parent and which was the child.  To convey that, I emphasized the part corresponding to the row where the relationship is stored.  That is, if parent/child is shown on the parent case, emphasize parent, and likewise for child.

The directionality is also implied by the nesting in this table, so maybe I should just drop it.  What do you think?

Reply:
I see, that makes a bit more sense now but I don't think there's enough info in the UI itself to know what it means. As you said, the table nesting also gives an indication of direction so I'd be in favour of dropping it.

I might even go so far as to split the identifier & type into separate named columns so that it's very clear to the user which is which. Alternately you could add hover text to on each: "Relationship Identifier" & "Relationship Type"

Reply:
Good thoughts, thanks - I like the hover text in particular.  I'm creating a ticket to have USH product review this for potential release, so I'll include these ideas there.